packages/react-magma-dom/src/components/NavTabs/NavTabs.tsx

```diff
@@ -77,6 +77,7 @@ export const NavTabs = React.forwardRef<
       orientation={orientation || TabsOrientation.horizontal}
       ref={ref}
       theme={theme}
+      tabIndex={0}
```

Comment:
I think this line is causing the focus to start on the Nav container, then each individual tab:
![chrome-capture-2022-10-8](https://user-images.githubusercontent.com/91160746/200671468-147e58c1-9448-4bf5-9a69-738bbdcff463.gif)

Is this okay for accessibility?

Reply:
I moved that `tabIndex` to `Drawer` instead, seems to help with isolating the use case of a nav element without effecting the entire `NavTabs` component.